src/Time/History.hpp

```diff
@@ -41,9 +41,9 @@ class History {
   using size_type = size_t;
 
   History() = default;
-  History(const History&) = delete;
+  History(const History&) = default;
```

Comment:
@wthrowe is there any reason not to allow a copy of the History?

Reply:
No.  I probably deleted it because it was a somewhat expensive operation and I didn't need it for anything.